requirements.txt

```
@@ -4,6 +4,7 @@ croniter >= 0.3.24, <1.0
 dask >= 2.15.0
 distributed >= 2.15.0
 docker >=3.4.1
+importlib_resources >= 3.0.0
```

This is a backport from 3.9, should make this a conditional requirement

```suggestion
importlib_resources >= 3.0.0; python_version < '3.9'
```